lib/config/public.yml

```diff
@@ -22,9 +22,9 @@ default:
   threads: 5
 
   # LOGGING
-  log_old_number: 4           # number of log files before beginning to erase
+  log_old_number: 1           # number of log files before beginning to erase
   log_size: 1024000           # size of log file in bytes
-  log_level: Logger::WARN     # change to INFO for more detail
+  log_level: Logger::INFO     # change to INFO for more detail
```

Comment:
This is a little confusing with the comment to the side. Was this accidentally committed after some debugging?

I think we should increase the size limit. 1MB will probably be filled pretty quickly, especially with more verbose logging levels. I don't really know what an ideal default is. I had set 32MB (32768000 for quick reference) in my example, but that was pretty arbitrarily chosen. I think it should be safe to assume people will have 64-128MB of disk space to spare for each collection's repository and have a solid history in the log files :man_shrugging:

Reply:
Oh whoops, I did intend to change it to INFO in order to actually log what it was logging previously when this wasn't hooked up, but forgot to change the comment, my bad.  We can increase it.  I don't know as that many people will ever actually look back at these logs but I don't think space is a particularly large concern, either.